src/scss/components/_progress-indicator.scss

```diff
@@ -11,79 +9,148 @@
   }
 }
 
-.#{prefix('progress-indicator')} {
-  @extend .openforms-card; // TODO -> syntax highlighting trips on #{prefix('card')}
-
+/**
+ * Desktop and mobile styling for the progress indicator element.
+ *
+ * The parent component (_app.scss) is responsible for re-arranging the order of
+ * elements depending on the viewport. The styles of this component are responsible for
+ * managing how it looks/behaves on mobile/non-mobile viewports:
+ *
+ * - on mobile, the element takes a navbar-like approach that can be expanded/collapsed
+ * - on desktop, it behaves like a sticky element, so that it can be placed in a sidebar
+ *
+ * TODO: remove the fallbacks/defaults after a deprecation period. Many of these
+ * design tokens here didn't exist before 2.1 and had hardcoded values.
+ */
+.openforms-progress-indicator {
+  @include card.card('progress-indicator');
+
+  position: sticky;
+  inset-block-start: var(
+    --of-progress-indicator-inset-block-start,
```

Comment:
they are newly added so that theme designers can influence these aspects of the theme/styling. We should set them in our own theme, but I now see that I may have forgotten to add/commit a file :thinking: I'll get back to that!

Reply:
design tokens PR is updated!